lib/timescaledb/rails/model.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Timescaledb
+  module Rails
+    # :nodoc:
+    module Model
+      PUBLIC_SCHEMA_NAME = 'public'
+
+      extend ActiveSupport::Concern
+
+      # :nodoc:
+      module ClassMethods
+        # Returns only the name of the hypertable, table_name could include
+        # the schema path, we need to remove it.
+        #
+        # @return [String]
+        def hypertable_name
+          table_name.split('.').last
+        end
+
+        # Returns the schema where hypertable is stored.
+        #
+        # @return [String]
+        def hypertable_schema
+          if table_name.split('.').size > 1
```

Comment:
schemas may have dots in them as well, please check that

Reply:
good catch, just pushed a change to cover that new use case, thanks